### ISSUE TITLE
[COMPLETE_RELEASED] SWARM-1317 — portable RESUME parent-checkpoint authority crossbind

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1317]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1317
INSTANCE_TAG: SOL-SCI-CLOSER-20260911T1816+02-RECOVERY-TRUST
STATUS: ACTIVE_PENDING_POSTCLAIM_ARBITRATION
SWARM_LANE_KEY: D05|PORTABLE-RESUME|PARENT-CHECKPOINT-AUTHORITY-CROSSBIND|V1
PREFERRED_LANE: #14
PREFERRED_WORK_KIND: REPRODUCIBILITY_RELEASE
SELECTED_WORK_KIND: INDEPENDENT_VERIFY / RECOVERY_AUTHORITY / PRE-STEP TRUST ROOT
PARENT_ISSUES: #6 #14 #548 #723 #813 #1248 #1251 #1256 #1317
LIVE_MAIN_AT_CLAIM: 7accc766d76693a69dabb5ff853cb35040ec4728
TARGET_PR: #1256
TARGET_HEAD: ab60bd80689a91522c00c95dd26009debe166251
INTEGRATION_OWNER_AT_CLAIM: #1238
CLAIMED_AT_UTC: 2026-09-11T16:22:30Z
LEASE_UNTIL_UTC: 2026-09-11T22:22:30Z

CANDIDATE COMPARISON:
1. Portable RESUME parent-checkpoint authority/lineage trust root — SELECTED: exact PR1256 path copies caller-supplied parent checkpoint lineage and authority; packet validation separately shape-checks hashes/flags/authority but does not cryptographically or externally bind the authority to that lineage.
2. Portable evaluation-schedule crossbind — YIELD #1311.
3. Optimizer/scheduler/precision/seed crossbind — YIELD #1304.
4. Backend/runtime authority trust root — YIELD released #1313 finding.
5. Central integration — YIELD #1238.

READ_ONLY SCOUT SIGNAL:
- `portable_run_binding._build_candidate()` copies `overlay.checkpoint.lineage` and `parent_checkpoint_authority` into the candidate packet.
- `portable_run_packet._resume_blockers()` requires shape-valid parent checkpoint/manifest SHA-256, previous_run_id, resume_validated=true, cross_provider_transfer=true, differing providers, and a generic terminal authority envelope.
- No observed check binds `authorities.parent_checkpoint` to the exact parent checkpoint SHA, manifest SHA, previous run, provider, or separately trusted D05 recovery authority.
- Existing positive cross-provider fixture locally fabricates `_authority()` plus arbitrary SHA64 lineage and still asserts `binding_ready`.

OWNED SURFACE:
- read-only exact-head diagnosis of portable RESUME parent-checkpoint authority -> lineage trust binding;
- determine whether coherent caller resealing can reach READY_FOR_CROSS_PROVIDER_RESUME while independently expected/trusted recovery evidence stays unchanged;
- minimum existing-lineage repair/handoff only.

AVOID SURFACES:
- no Product mutation, branch, PR, merge;
- no #1251 readiness mutation, #1248 bounded-pilot mutation, D05 core mutation, D06 mutation, recipe mutation or integration action;
- no evaluation-schedule semantic (#1311), optimizer/seed semantic (#1304), backend authority semantic (#1313), or session-overlay scalar/type semantic (#1312);
- no data/tokenizer/model-training/final-test/paid-compute surface.

ACCEPTANCE:
1. direct paginated OPEN-collection arbitration after this claim finds no earlier valid same-semantic owner;
2. late-bind exact main/PR1256 head before verdict;
3. verify exact positive RESUME construction accepts caller-selected coherent parent lineage plus generic terminal-shaped parent authority;
4. verify neither binding nor packet validator authenticates parent authority against exact lineage or an out-of-packet D05 recovery trust root;
5. if another consumed trusted binding exists before optimizer step 1, return NO_NEW_BLOCKER; otherwise publish exact pre-step finding and minimal repair direction;
6. handoff to #1251/#1248/#1238 as appropriate, without self-certification or merge.

TRUTH_BOUNDARY:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_EXECUTED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

**Comment by Oleksii-debug** [comment]
## TERMINAL READ-ONLY VERDICT — CONCRETE_CROSSBIND_FINDING

`RECOVERYTRUST1317-001 — HIGH PRE-STEP — portable RESUME parent-checkpoint authority is not bound to the exact recovery lineage`

### Late-bound exact state
- live Product surface rebased during this audit from PR #1256 to newer PR #1316;
- exact current audited Product head: `29d4d73f9ae2701d7d034574ff0065530ef9366d`;
- exact shared CI: `34621603816 = SUCCESS`;
- PR #1316 is open/mergeable and changes only portable session-overlay scalar/type validation + tests; the recovery trust semantic below remains unchanged;
- current `portable_run_packet.py` blob on that exact head remains `2fd6f437df385206ef4575f949759d9c22014351`.

### Postclaim arbitration
Direct OPEN collection was exhaustively enumerated with `per_page=100`, pages 1..8; page 9 is empty. Exact lane key and exact `parent_checkpoint_authority` semantic appear only in #1317. #1318 audits PR #1316 scalar/type repair and explicitly avoids recovery semantics. Arbitration: **WIN**.

### Exact evidence
1. `portable_run_binding._build_candidate()` copies `overlay.checkpoint.lineage` verbatim into `packet.checkpoint.lineage` and copies `overlay.checkpoint.parent_checkpoint_authority` verbatim into `packet.authorities.parent_checkpoint`.
2. `_coherence_blockers()` binds code/model/backend only. It has no parent-checkpoint/lineage coherence check and consumes no independently expected D05 recovery root.
3. `portable_run_packet._valid_terminal_authority()` validates only generic envelope shape: repository, git SHA shape, evidence SHA shape, positive workflow run id, `workflow_conclusion=success`, `terminal=true`.
4. `_resume_blockers()` independently shape-checks `parent_checkpoint_sha256`, `parent_manifest_sha256`, nonempty `previous_run_id`, `resume_validated=true`, `cross_provider_transfer=true`, source/target provider difference, then separately calls the generic authority-shape check. No value from the authority authenticates any lineage field.
5. Exact-head positive regression `test_ready_cross_provider_resume_binds_parent_lineage()` constructs caller-selected `SHA64` parent checkpoint + manifest, `R01-SESSION-001`, `OWNER_LAPTOP`, sets `resume_validated=true`, and fabricates `parent_checkpoint_authority = _authority()`; with unrelated readiness tokens valid it asserts `result.binding_ready` and `mode == RESUME`.

### Consequence
Once unrelated readiness gates become terminal, a coherent caller can substitute/reseal the parent checkpoint SHA, parent manifest SHA, previous-run ID/source provider and `resume_validated` assertion while supplying any terminal-shaped parent authority envelope. The portable boundary can therefore report cross-provider RESUME-ready without proving that the exact checkpoint/manifest/run/provider tuple was independently D05-qualified. Current exposure is still zero, so this is a latent pre-step trust-root defect, not evidence that invalid optimizer updates already occurred.

### Minimum same-lineage repair
Do not create another checkpoint framework. At the existing portable binding boundary, consume an independently reconciled D05 recovery/checkpoint authority that is metadata-bound to at least the exact parent checkpoint SHA, parent manifest SHA, previous run identity, source provider and validated-resume outcome. A fixed trusted authority/binding must fail if any of those candidate-side fields change, even under coherent candidate resealing. Keep source/target provider policy and scalar/type checks separate.

Required adversarial regression: start from an intended-positive RESUME packet with one externally trusted parent-checkpoint binding; mutate each lineage field one at a time while keeping the trusted binding fixed; every mutation must fail closed. Also reject a newly self-authored authority/lineage pair when the independently expected recovery root is unchanged.

### Scientific truth boundary
`AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0`; no tokenizer fit, optimizer update, model training, learned weights, final-test outcome access, paid compute, foreign pretrained weights, or external LLM/API corpus intelligence occurred in this audit.

**Comment by Oleksii-debug** [comment]
## RELEASE
STATUS: COMPLETE_RELEASED_READ_ONLY
LEASE_RELEASED_EARLY=true
OWNERSHIP_RELEASED=true
PRODUCT_MUTATION=false
BRANCH_CREATED=false
PR_CREATED=false
MERGE_PERFORMED=false

Delivered: terminal finding `RECOVERYTRUST1317-001`, Product handoff #1312 comment 5637465748, readiness handoff #1251 comment 5637467001, bounded-pilot handoff #1248 comment 5637468373, integration handoff #1238 comment 5637469698.

NEXT: refresh live main/claims and rotate this same SWARM-1317 registration to the next highest-value collision-free learned-model blocker. Do not retain this recovery ownership.
